class/defaults.yml

```diff
@@ -14,3 +14,8 @@ parameters:
         repository: docker.io
         image: ongres/kubectl
         tag: v1.24.3-build-6.16
+    helmParams:
```

Comment:
side-note: I probably wouldn't expose the whole `helmParams` as a component parameter, but that's your call.

Reply:
Refactored it accordingly.